backend/pick-git/src/test/java/com/woowacourse/pickgit/config/DatabaseCleaner.java

```diff
@@ -0,0 +1,62 @@
+package com.woowacourse.pickgit.config;
+
+import org.hibernate.Session;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.stereotype.Component;
+import org.springframework.test.context.ActiveProfiles;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+@ActiveProfiles("test")
+public class DatabaseCleaner implements InitializingBean {
+
+    @PersistenceContext
+    private EntityManager entityManager;
+
+    private List<String> tableNames;
+
+    @Override
+    public void afterPropertiesSet() {
+        entityManager.unwrap(Session.class)
+                .doWork(this::extractTableNames);
+    }
+
+    private void extractTableNames(Connection conn) throws SQLException {
+        List<String> tableNames = new ArrayList<>();
+
+        ResultSet tables = conn
+                .getMetaData()
+                .getTables(conn.getCatalog(), null, "%", new String[]{"TABLE"});
+
+        while(tables.next()) {
+            tableNames.add(tables.getString("table_name"));
+        }
+
+        this.tableNames = tableNames;
+    }
+
+    public void execute() {
+        entityManager.unwrap(Session.class)
+                .doWork(this::cleanUpDatabase);
+    }
+
+    private void cleanUpDatabase(Connection conn) throws SQLException {
+        Statement statement = conn.createStatement();
+        statement.executeUpdate("SET REFERENTIAL_INTEGRITY FALSE");
+
+        for (String tableName : tableNames) {
+            statement.executeUpdate("TRUNCATE TABLE " + tableName);
+            statement.executeUpdate("ALTER TABLE " + tableName + " ALTER COLUMN ID RESTART WITH 1");
```

Comment:
```suggestion
            statement.executeUpdate("ALTER TABLE " + tableName + " ALTER COLUMN id RESTART WITH 1");
```

컬럼명은 소문자로 작성하면 좋을 거 같아요 !